src/shared/configs/jwt.ts

```diff
@@ -1,15 +1,15 @@
 if (!process.env.JWT_SECRET) {
   throw new Error('process.env.JWT_SECRET is undefined');
 }
-if (!process.env.JWT_ACCESS_LIFETIME) {
-  throw new Error('process.env.JWT_ACCESS_LIFETIME is undefined');
+if (!process.env.JWT_ACCESS_DEATHDATE) {
+  throw new Error('process.env.JWT_ACCESS_DEATHDATE is undefined');
 }
-if (!process.env.JWT_REFRESH_LIFETIME) {
-  throw new Error('process.env.JWT_REFRESH_LIFETIME is undefined');
+if (!process.env.JWT_REFRESH_DEATHDATE) {
+  throw new Error('process.env.JWT_REFRESH_DEATHDATE is undefined');
 }
 
 export const jwtConfig = {
   secret: process.env.JWT_SECRET,
-  accessLifetime: Number(process.env.JWT_ACCESS_LIFETIME),
```

Comment:
It's better to cast to Number earlier. What if the record in `.env` exists but it's getting transformed into `NaN`?